app/controllers/concerns/form_attachment_create.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module FormAttachmentCreate
+  extend ActiveSupport::Concern
+
+  included do
+    skip_before_action(:authenticate)
+  end
+
+  def create
+    form_attachment_model = self.class::FORM_ATTACHMENT_MODEL
+    form_attachment = form_attachment_model.new
+    namespace = form_attachment_model.to_s.underscore.split('/').last
+    form_attachment.set_file_data!(params[namespace][:file_data])
+    form_attachment.save!
+    render(json: form_attachment)
```

Comment:
Will this response contain the full encoded file data? Is the size of that response a concern? I guess load testing will give us a solid answer on this. Otherwise, LGTM.

Reply:
@rlbaker no this is the PR for putting images into save in progress https://github.com/department-of-veterans-affairs/vets-api/pull/1663